File: python/bob/bob.py
def response(hey_bob):
    copy=hey_bob.strip()
    if copy.isupper():
        return "Calm down, I know what I'm doing!" if copy.endswith('?') else 'Whoa, chill out!'   
    return 'Sure.' if copy.endswith('?') else "Fine. Be that way!" if copy=="" else "Whatever."

File: javascript/bob/bob.js
//
// This is only a SKELETON file for the 'Bob' exercise. It's been provided as a
// convenience to get you started writing code faster.
//
function isCaps(message) {
  return message.match(/[A-Z]/) && message===message.toUpperCase()
}
export const hey = (message) => {
    const trimmedMessage = message.trim(), isQuestion = trimmedMessage.endsWith('?'), isEmpty = trimmedMessage.length === 0
    if (isEmpty) return "Fine. Be that way!";
    if (isCaps(trimmedMessage)) return isQuestion ? "Calm down, I know what I'm doing!" : 'Whoa, chill out!';
    if (isQuestion) return 'Sure.';
    return 'Whatever.';
};
